src/utils/string/jiraLinkify.tsx

```diff
@@ -4,10 +4,15 @@ import { getJiraTicketUrl } from "constants/externalResources";
 
 export const jiraLinkify = (
   unlinkified: string | React.ReactNodeArray,
-  jiraHost: string
+  jiraHost: string,
+  onClick?: (match: string) => () => void
 ) =>
   reactStringReplace(unlinkified, /([A-Z]{1,10}-\d{1,6})/g, (match, i) => (
-    <StyledLink key={`${match}${i}`} href={getJiraTicketUrl(jiraHost, match)}>
+    <StyledLink
+      onClick={onClick ? onClick(match) : undefined}
```

Comment:
we can assign a default value to the onClick callback above and avoid this ternary.

Reply:
I think you missed this comment.